source/getting-started/welcome_email.rst

```diff
@@ -0,0 +1,55 @@
+Welcome Email to End Users
+===========================
+
+
+To make it easy for your end users to start using Mattermost right away, we created a sample email template that you can use.
+
+Remember to replace all the items below in bold with your information.
+
+
+Email Template
+-------------------
+
+From: **[company name]** IT Team
+
+To: End users
+
+Subject: New Communication Platform - Mattermost
+
+
+Hi all,
+ 
+As some of you already know, we are moving to Mattermost as our communication platform. Mattermost is a messaging app where you can talk, share files, and collaborate on projects or initiatives. Mattermost also integrates with many of the apps that you use every day, like **[add apps]**.  
+
+We are moving to Mattermost because it will allow us to become more productive by being able communicate in one place instead of using email, instant chat, and a multitude of other programs.
+
+Some of the major benefits of using Mattermost are:
+
+- Direct 1:1 and group messaging
+
+- Channels for topic-based, group-based, or meeting-based chat
+
+- Streamlined collaboration on projects
+
+- Reduced email clutter
+
+- Searching across messages and channels
+
+- Sharing files
+
+To get started:
+
+1. Open a browser on your computer, go to **[Mattermost URL]** and log in with your **[LDAP/AD, SAML, Google, etc]** credentials. 
+
+2. `Download <https://about.mattermost.com/download/#mattermostApps>`_ the Mattermost apps for desktop and mobile. Refer to the `Mattermost Welcome Guide <https://docs.mattermost.com/help/getting-started/welcome-to-mattermost.html>`_ on how to get up and running quickly.
```

Comment:
Should this step be optional?

Reply:
Desktop and mobile are a big part of our customers' every use case. They can skip if they don't want to download. The welcome guide is essential to them getting up and running quickly on MAttermost.